### PR TITLE
Changing owner and permissions on /var/cache/icinga2

### PR DESCRIPTION
**Pull request by WingsLikeEagles**
This should fix the issues when running on a Windows host.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -139,6 +139,9 @@ RUN true \
     && mkdir -p /var/log/icinga2 \
     && chmod 755 /var/log/icinga2 \
     && chown nagios:adm /var/log/icinga2 \
+    && mkdir -p /var/cache/icinga2 \
+    && chmod 755 /var/cache/icinga2 \
+    && chown nagios:adm /var/cache/icinga2 \
```
Comment (jjethwa) [inline Dockerfile:144]: Ownership looks to be nagios:nagios for /var/cache/icinga2

**Review by jjethwa — CHANGES_REQUESTED** [review]

**Comment by jjethwa** [comment]
Hey @WingsLikeEagles 

Thanks so much for the PR!  I've requested one small update.

**Comment by jjethwa** [comment]
Reference: https://github.com/jjethwa/icinga2/issues/292

**Comment by WingsLikeEagles** [comment]
Yeah, go for it.  I was just copying line 141 which had nagios:adm

**Comment by WingsLikeEagles** [comment]
Ok, should be good.  Sorry for the delay.  I don't see notifications.

**Comment by jjethwa** [comment]
Thanks, @WingsLikeEagles 

I'm on vacation right now, but will build and push the new image as soon as I get back. Thanks again 😃

**Pull request merged**